apps/design/frontend/src/ballots_screen.test.tsx

```diff
@@ -131,28 +132,26 @@ describe('Ballot styles tab', () => {
     let modal = await screen.findByRole('alertdialog');
     within(modal).getByRole('heading', { name: 'Confirm Finalize Ballots' });
     userEvent.click(within(modal).getByRole('button', { name: 'Cancel' }));
-    await waitFor(() =>
+    await vi.waitFor(() =>
       expect(screen.queryByRole('alertdialog')).not.toBeInTheDocument()
     );
 
-    jest.useFakeTimers();
+    vi.useFakeTimers();
     const finalizedAt = new Date();
-    jest.setSystemTime(finalizedAt);
-    apiMock.setBallotsFinalizedAt
-      .expectCallWith({ electionId, finalizedAt })
-      .resolves();
+    vi.setSystemTime(finalizedAt);
```

Comment:
Now that the timestamp is captured on the backend, we can remove fake timers from this test